src/solver.js

```diff
@@ -1357,7 +1371,9 @@ Tableau.prototype.applyCuts = function (branchingCuts){
 Tableau.prototype.branchAndCut = function () {
     var branches = [];
     var iterations = 0;
-
+    var optcr = this.optcr;
+    var relaxedSolution = this.relaxedSolution * (1 - (optcr/100));
```

Comment:
I would name this variable differently because it does not exactly correspond to the relaxed evaluation. I would choose a name that correspond to how we want to use it in this context, may be ```acceptableThreshold```?